packages/google-cloud/.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  env: {
+    node: true,
+  },
+  extends: ['../../.eslintrc.js'],
+  rules: {
+    '@sentry-internal/sdk/no-optional-chaining': 'off',
```

Comment:
l: I think this is by now disabled by default, so we don't need to overwrite this anymore.